README.md

```diff
@@ -1 +1,81 @@
 # server-Amazin
+
+# client-Amazin
+
+This is the server (back-end) side of our Amazin' project created by Talent Path's 'Quaran-Team!' of full stack developers.
+
+<!-- Picture below of home page of our Amazin' App  -->
+
+![alt text](https://github.com/Quaran-Team/client-Amazin/blob/master/amazinscreen.JPG "Screen shot from the Amazin' App mock e-commerce site")
+
+---
+
+The plan was to practice weekly agile sprints by creating an e-commerce product page(s) for 20-50 items in a very similar theme to that big one named after the giant river in South America.
+
+Each team member has made contributions to both client (front - end) and server (back - end) sides of the project connecting their data to their components.
+
+The client side can be found here _-_ GitHub Repo = https://github.com/Quaran-Team/client-Amazin .
+
+And live deployed at ... {}.
+
+---
+
+We created the server in Java using Spring-Boot. Using Maven for our dependicies and Fly Way to manage our data migrations. 
+
+Due to time constraints some parts are just static mocks, while others are fully dynamic.
+
+to see it live
+[link](www.wherewhenever we deploy. com)
+
+---
+
+To run it locally:
+1- clone down the repo
+2- run the program either in an ide (we used either intelli-J or vs code) by clicking the little green play button arrow for the file ApiApplication.java ... or via command line with mvn spring-boot:run
+
+If updates are made to the data sql files run the commands 'mvn flyway:clean' followed by 'mvn clean' before starting the server.
+
+If you get an error command not found MVN you will need to download and install it locally which can be done here: https://maven.apache.org/download.cgi
+---
```

Comment:
probably mention it would be good to do run ` mvn (or whatever command maven has you use) install ` to make sure you get the plugins. `mvn clean install` works as well to make sure there are no issues the first time.

also mention `mvn spring-boot:start` can be used to start the application once inside api folder